src/media/modules/class-wpml-elementor-media-node-image-gallery.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+class WPML_Elementor_Media_Node_Image_Gallery extends WPML_Elementor_Media_Node {
```

Comment:
@strategio This class and the previous three are exactly the same except for the property name. Can you remove the code duplication.

Reply:
Done @brucepearson, I created some abstract classes to reuse for nodes with the same pattern.